beatrix/src/test/java/org/killbill/billing/beatrix/integration/usage/TestConsumableInArrear.java

```diff
@@ -535,10 +535,14 @@ public void testWithFullBlockBilling() throws Exception {
 
         // 2012-04-21 -- block billing effective date is 2012-04-01
         clock.addDays(20);
-        busHandler.pushExpectedEvents(NextEvent.BLOCK);
+        busHandler.pushExpectedEvents(NextEvent.BLOCK, NextEvent.INVOICE);
         final BlockingState blockingState1 = new DefaultBlockingState(bpSubscription.getBundleId(), BlockingStateType.SUBSCRIPTION_BUNDLE, "state1", "Service", true, true, true, null);
         subscriptionApi.addBlockingState(blockingState1, new LocalDate(2012, 4, 1), ImmutableList.<PluginProperty>of(), callContext);
         assertListenerStatus();
 
+        invoiceChecker.checkInvoice(account.getId(), 3, callContext,
+                                    new ExpectedInvoiceItemCheck(new LocalDate(2012, 5, 1), new LocalDate(2013, 5, 1), InvoiceItemType.REPAIR_ADJ, new BigDecimal("-2399.95")),
```

Comment:
There's no repair on the `2012-04-1` - `2012-04-30` period because it's $0?

Reply:
Yep, just a `FIXED` price and regardless of amount, we don't repair `FIXED` items.